js/start.js

```diff
@@ -37,6 +37,7 @@ import { fetchExchangeRates } from './utils/currency';
 import './utils/exchangeRateSyncer';
```

Comment:
Can you please make sure to lint this PR and fix any lint errors? `npm run lint`

Reply:
Fixed, thanks. 
Could Travis be used to run this automatically? Currently it fails for PRs because it tries to build distribution packages but doesn't have the secrets needed to do that when running from a PR. Maybe you could run lint and tests on PRs and full build on tags only.